docs/build/iota-sdk/2.0/docs/getting-started/wasm.mdx

```diff
@@ -0,0 +1,207 @@
+---
+sidebar_label: Wasm
+description: Getting started with the IOTA Identity WASM Library.
+image: /img/Identity_icon.png
+tags:
+  - WASM
+  - install
+  - npm
+  - yarn
+  - build
+  - nodejs
+  - webpack
+---
+import WarningPasswordStorage from '../_admonitions/_warning-password-storage.md';
+import EditDotEnv from '../_admonitions/_edit_dot_env.md';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Getting Started with WASM
+
+<WarningPasswordStorage />
+
+## Which Bindings to Choose?
+
+The IOTA SDK library also offers dedicated [Node.js bindings](../nodejs). The differences with this package are outlined
+below.
+
+**NOTE: Use the Node.js bindings if you can. The Wasm bindings are more portable and exist to support browser
```

Comment:
Duplicate. Do we want to mention it twice?

Reply:
```suggestion
```